airflow/providers/apache/hive/CHANGELOG.rst

```diff
@@ -27,6 +27,17 @@
 Changelog
 ---------
 
+7.0.0
+.....
+
+
+Breaking changes
+~~~~~~~~~~~~~~~~
+
+* ``use_beeline``, ``proxy_user``, ``principal`` in hive client connection options is moved from the extra field to ``Use Beeline``, ``Proxy User``, ``Principal`` parameter in the Hook.
```

Comment:
This item is not a braking change?
It just how it is visualized in the connection form (sperated field) nothing is changed in terms of functionality

Reply:
removed